symbolic-unwind/src/evaluator/parsing.rs

```diff
@@ -399,18 +399,28 @@ mod test {
     #[test]
     fn test_expr_1() {
         let input = "1 2 + 3 *";
-        let e = Expr::Op(
-            Box::new(Expr::Op(
-                Box::new(Expr::Value(1u8)),
-                Box::new(Expr::Value(2)),
-                BinOp::Add,
-            )),
-            Box::new(Expr::Value(3)),
-            BinOp::Mul,
-        );
-        let (rest, parsed) = expr(input).unwrap();
+        let (rest, parsed) = expr::<u8>(input).unwrap();
         assert_eq!(rest, "");
-        assert_eq!(parsed, e);
+        insta::assert_debug_snapshot!(
```

Comment:
Try out with `ron`, this could compact the snapshot a bit.
https://docs.rs/insta/1.7.1/insta/macro.assert_ron_snapshot.html